### ISSUE TITLE
Using Redux Observable to handle async leads to actions being fire twice

### ISSUE THREAD

**Issue by walleXD** [hardchor/electron-redux#67]
Hey,

I was trying to use `redux-observable` but the `synchronous` action gets fired by the `async` action gets fired twice. Any idea why that would be happening?

Thanks a bunch 🙏🏼

**Comment by hardchor** [comment]
Hey @walleXD 

That's probably happening because your async action fires both on the renderer and the main thread. Have you tried [aliased actions](https://github.com/hardchor/electron-redux#aliased-actions-main-process) ?

**Comment by walleXD** [comment]
I am just firing the actions in the renderer. Should I scope them with `local` to avoid being run twice?

Aliased actions allow us to fire the action in the main thread but this happens in the renderer but I didn’t try to use aliased action On the main thread with redux-observable. Will report back in my findings.

**Comment by hardchor** [comment]
Yeah, so depending on what you're doing, your option is to either use the `local` option and prevent the action from being forwarded to the main thread's store, or use an aliased action to only trigger the action in the main store (which is the preferred way for actions that produce side-effects, e.g. AJAX requests etc.).
